flavors/standard-EXASOL-7.1.0-without-python2.7/flavor_base/flavor_base_deps/packages/python3_pip_packages

```diff
@@ -41,4 +41,5 @@ pykickstart|3.33
 pysftp|0.2.9
 pytz|2021.1
 git+https://github.com/numpy/numpy.git@maintenance/1.19.x
-sagemaker|2.59.5
\ No newline at end of file
+sagemaker|2.59.5
+grafeas|1.4.0
```

Comment:
Maybe we should mention the addition of this package to 7.1 in the release notes?